config/routes.rb

```diff
@@ -1,10 +1,13 @@
 Rails.application.routes.draw do
   root "photos#index"
 
+  get "/users/:id" => "users#show", as: :user
```

Comment:
Please delete or comment out this line since we are using the route on line 12 for usernames